hicommonwealth/commonwealth
# 9710 adjustable connector weight

## Link to Issue
Closes: #9710

## Description of Changes
- adds variable connector weight and new contracts with updated signature

## Test Plan
- Deploy new token 
- ensure deploy + buy/sell still work

## Other Considerations

- Only new tokens will work from this point on
